bin/test-cleanup

```diff
@@ -37,69 +19,11 @@ echo "cleaning up multicluster resources, if present [${k8s_context}]"
 echo "cleaning up jaeger extension resources, if present [${k8s_context}]"
 "$linkerd_path" jaeger uninstall 2> /dev/null | kubectl --context="$k8s_context" delete -f -
 
-# Initialize empty arrays
-namespaces_controlplane=()
-namespaces_cni=()
-namespaces_dataplane=()
-clusterrolebindings=()
-clusterroles=()
-webhookconfigs=()
-validatingconfigs=()
-podsecuritypolicies=()
-customresourcedefinitions=()
-apiservices=()
-rolebindings=()
-
-echo "cleaning up control-plane namespaces in k8s-context [${k8s_context}]"
-
-populate_array namespaces_controlplane   ns                              is-control-plane 'control-plane namespaces'
-
-echo "cleaning up CNI namespaces in k8s-context [${k8s_context}]"
-
-populate_array namespaces_cni ns         cni-resource                    'CNI namespaces'
+echo "cleaning up the test resources via linkerd uninstall"
```

Comment:
Nit, but these aren't exactly test resources; this is the control plane being uninstalled 🙂

```suggestion
echo "cleaning up linkerd resources [${k8s_context}]"
```

Reply:
ahh my bad. 🙈